server/AccManagement.java

```diff
@@ -0,0 +1,50 @@
+package newbank.server;
+
+public class AccManagement{
```

Comment:
As discussed, this should be renamed, some candidates are AccountManagement, Accounts, Account..